### ISSUE TITLE
multilevel correlation for varying slopes

### ISSUE THREAD

**Issue by btcrewther** [easystats/datawizard#203]

**Comment by bwiernik** [comment]
The multilevel argument itself is misleadingly named IMO. I would expect it to decompose correlations into within and between components. Instead, it adjusts for person factors, either with or without partial pooling. I don't really understand the use for the results it currently reports.

**Comment by DominiqueMakowski** [comment]
It's mostly used in the case of multilevel correlations afaik, with the common usecase being to somewhat take into account some grouping factors. Obviously not ideal, but can be useful in exploratory analyses (and I've heard via various sources that it could be quite a popular feature of *correlation*).

 In that context, having a "full" random effect specification `y ~ x + (x | g)` is not unreasonable i think

**Comment by bwiernik** [comment]
> It's mostly used in the case of multilevel correlations

I don't know what you mean. When I hear "multilevel correlation", I think the within/between correlations computed by eg https://www.rdocumentation.org/packages/psych/versions/2.2.5/topics/statsBy

What we currently compute is neither within nor between

**Comment by DominiqueMakowski** [comment]
right, I meant correlation partialized via mixed/multilevel models (https://github.com/easystats/correlation#multilevel-correlations)

**Comment by bwiernik** [comment]
What I'm trying to say is that I don't understand what quantity such a estimator is estimating

**Comment by DominiqueMakowski** [comment]
it then converts the coefficients to "partialized" correlations, and is used as such

**Comment by IndrajeetPatil** [comment]
Why is this back in _datawizard_, @DominiqueMakowski? I thought this was about correlation analysis?

**Comment by DominiqueMakowski** [comment]
Adjust was always in datawizard
